frame/ethereum/src/lib.rs

```diff
@@ -302,6 +313,16 @@ impl<T: Trait> Module<T> {
 		frame_system::Module::<T>::deposit_log(digest.into());
 	}
 
+	fn logs_bloom(logs: Vec<Log>, bloom: &mut Bloom) -> Bloom {
```

Comment:
Is it common to mutate the value and also return it? Also a dumb Rust question. When we return `*bloom` is that  move or a copy?

Reply:
> Is it common to mutate the value and also return it?
Just a way of assigning the field values without having to define the mutable variable, but I guess at the cost of readability I guess.. changed it.

> When we return *bloom is that move or a copy?
If Bloom implements the Copy trait (I believe it doesn't), it will be a copy. Otherwise move.